qdrant_client/local/distances.py

```diff
@@ -147,6 +147,14 @@ def calculate_distance_core(
         return calculate_distance(query, vectors, distance_type)
 
 
+def fast_sigmoid(x: np.float32) -> np.float32:
+    if np.isfinite(x):
+        return x / (1.0 + abs(x))
+    else:
```

Comment:
Small thing:

You can remove the else and directly do return X no?

Reply:
Or https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.expit.html ?

Reply:
True, we can avoid the redundant else statement. 

> expit

We use the same function as in core implementation to get scoring congruence

Reply:
Oh I see